{{cookiecutter.project_repo}}/pyproject.toml

```diff
@@ -0,0 +1,58 @@
+[tool.poetry]
+name = "{{ cookiecutter.project_slug }}"
+version = "{{ cookiecutter.version }}"
+authors = [
+    "UK Hydrographic Office <datascienceandengineering@ukho.gov.uk>"
+]
+description = "Python wrapper for the C implementation of the Generic Sensor Format library."
```

Comment:
I think there is a "{{ }}" variable for the description

Reply:
Fixed